stdlib/Test/src/Test.jl

```diff
@@ -1153,14 +1190,13 @@ function print_test_results(ts::DefaultTestSet, depth_pad=0)
     fail_width   = dig_fail   > 0 ? max(length("Fail"),   dig_fail)   : 0
     error_width  = dig_error  > 0 ? max(length("Error"),  dig_error)  : 0
     broken_width = dig_broken > 0 ? max(length("Broken"), dig_broken) : 0
-    total_width  = dig_total  > 0 ? max(length("Total"),  dig_total)  : 0
-    duration_width = max(length("Time"), length(duration))
+    total_width  = max(length("Total"),  dig_total)
```

Comment:
This didn't stick in my review it seems.
Just a minor suggestion. I assume NFC but for consistency with the surrounding code.
```suggestion
    total_width  = max(textwidth("Total"),  dig_total)
```